### PR TITLE
Merging recent Code and Document changes

### PR DESCRIPTION
**Pull request by JeremySpradlin**
- Updated documentation to include cleaning up comments, etc
- Included Design Doc in Notebooks/ 
- Removed excess files/code
- Added DB Object classes 'Judge' and 'Case'

### CODE REVIEW

**Review by fansha1994 — APPROVED** [review]
Great work!!

**Comment by serinamarie** [comment]
It looks like this also removes the Procfile and Dockerfile used for EB deployment. Is there documentation that this is deploying to Heroku rather than AWS?

**Comment by serinamarie** [comment]
There is also Beanstalk listed in the tools rather than Heroku

**Comment by JeremySpradlin** [comment]
@serinamarie Documentation is there referencing the issues with deployment, but the docker and Procefile were removed or modified during the process of troubleshooting the deployment to beanstalk.  Those came from Ryan Herr.

**Comment by serinamarie** [comment]
Sounds good, still waiting on the README update from Moe regarding an update on dev tools used.

File: README.md [hunk 1]
```diff
@@ -7,26 +7,37 @@ An application to assist immigration advocates in winning asylum cases
...
 
+* After converting the uploaded PDF to text, we will extract relvant entities from the text to store relevant features in a DB to facilitate data recall and analysis
+
+
 ## Tools
 
+ * [Python 3](https://www.python.org/downloads/)
  * [Pytesseract](https://github.com/madmaze/pytesseract)
  * [FastAPI](https://github.com/tiangolo/fastapi)
  * [AWS Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/)
```
Comment (serinamarie) [inline README.md:26]: I would recommend removing this

**Comment by fansha1994** [comment]
@serinamarie @JeremySpradlin  just added the link for Heroku tutorial to the readme.md file with a new pull request

**Pull request closed without merging**